operator-framework-core/src/main/java/io/javaoperatorsdk/operator/api/config/Version.java

```diff
@@ -0,0 +1,45 @@
+package io.javaoperatorsdk.operator.api.config;
+
+import java.util.Date;
+
+/** A class encapsulating the version information associated with this SDK instance. */
+public class Version {
+
+  private final String project;
+  private final String commit;
+  private final Date builtTime;
+
+  public Version(String project, String commit, Date builtTime) {
+    this.project = project;
```

Comment:
@metacosm still not sure about why is this? this seems to me some opionated way. Especially with build time and project. Commit id is fine? Is there some deeper meening of this, or its required by quarkus?

Reply:
@csviri Maybe a better name would be helpful, indeed. This corresponds to the Maven project version. Why would it be redundant? The goal is to make sure that the operator is running the SDK version the user thinks it's running which can be helpful to diagnose issues.

Reply:
I see, makes sense, but isn't the commit id enough for that, like log "running sdk version: {{commit id}}" . Or rather even the release version, thus version of the dependency.

Reply:
I think the project version is useful as a first check because it's not as easy to figure out which version of the SDK you're running solely from the commit id.

Reply:
Absolutely, I agree, I just not see any field for real version (in terms of semantic versioning), just commit id and time stamp. Or am I missing something?

Reply:
Why wouldn't the sdk version be useful? If you have a running operator on a cluster, it's easier to look at the logs than try to figure out which version of the code was used to build the image. Or make sure that the image is actually running the version of the code you think it is… 😄

Reply:
I think its useful probably just the naming was confusing, so by "project" you meant the SDK Version right? I think that was the only issue, thus was not obviouse what it is.

Reply:
Yeah, I renamed it to `getSdkVersion`